src/metrics.rs

```diff
@@ -2,6 +2,8 @@ use std::{collections::HashMap, fmt, sync::RwLock};
 use tracing::{field::Visit, Subscriber};
 use tracing_core::{Field, Interest, Metadata};
 
+#[cfg(feature="otel_unstable")]
```

Comment:
Nit: please use spaces around `=` in attributes (for all of the added guards).

Reply:
I'm sure rustfmt can handle this. I raised a new issue at https://github.com/tokio-rs/tracing-opentelemetry/issues/143 and is working a PR to fix it.

Reply:
[fixed](https://github.com/tokio-rs/tracing-opentelemetry/pull/129/commits/25e7756839d2d61f47e1f736567cb09c4c2ce87e)